### DISCUSSION TITLE
No header in options!!!

### DISCUSSION THREAD

**Discussion by kravchenko-anton** [software-mansion/react-native-screens#2209] [Q&A]
Im using @react-navigation/native-stack and want migrate to your library. But in the Stack.Screen i using options: {{
			header: ({ navigation }) => (
				<Header.Head>
					<Header.Logo className='pl-2' />
					<Header.Icon
						className='pr-2'
						icon={SearchIcon}
						onPress={() => navigation.navigate('Search')}
					/>
				</Header.Head>
			),
			headerShown: true
		}}
		
		
		Why your library not provide this props in options?

**Comment by tboba** [comment]
Hi @kravchenko-anton! Currently, `header` prop does not exist in react-native-screens's native-stack, since that's the prop that only exists on native-stack from @react-navigation and has been introduced in native-stack v6. It's fully rendered on the JS side, which means that even you could render your own header and just set `headerShown` to false (you can see [here](https://github.com/react-navigation/react-navigation/blob/main/packages/native-stack/src/views/NativeStackView.native.tsx#L460) how react-navigation does that).

Also, I would **hardly** not recommend moving to the native stack from react-native-screens, since it's deprecated and shouldn't be used, as it might be removed soon. Is there any reason why do you want to use our native-stack? Please keep in mind that the native stack that comes from our library is marked as `v5` version, whereas in react-navigation you can find `v6` and `v7` parts of it 😄 

Since I think I've exhausted the topic, let me close this discussion! If I can do anything more for you, just ask me to reopen this discussion.
Cheers!
